### DISCUSSION TITLE
Traeger aquasission

### DISCUSSION THREAD

**Discussion by Minxster** [apption-labs/meater-cloud-public-rest-api#22] [General]
Hi, just wanted to ask... Any idea if the Traeger aquasission will cause problems for the API?

**Comment by nojacko** [comment]
Hey. There's no plans to change the API.

**Reply by Minxster** [reply]
Thanks @nojacko for getting back to me... When you say "no plans to change the API", do you mean "we're going to continue with this", or "we're in limbo, so we have no plans to make future changes to the project".

... I know its probably/realistically too early to say with 100% certainty either way, as it will depend on future event and changes to the business. I'm just sat her coding C++ for an ESP32 and it's all predicated on this API, so I'm trying to figure out if I should stop now or keep on going :-D :-)

**Reply by nojacko** [reply]
There's no limbo and we plan to continue supporting the API for the foreseeable future. To quote the official [press release](https://www.prnewswire.com/news-releases/traeger-grills-bolsters-connected-cooking-capabilities-with-meater-acquisition-301325708.html) "MEATER will operate as a standalone company within Traeger Grills".

**Reply by Minxster** [reply]
Brilliant @nojacko, thanks for the update/info. :-)
